### PR TITLE
Add local search functionality

### PR DESCRIPTION
**Pull request by zsherman**

### CODE REVIEW

**Comment by jsonmaur** [comment]
@zsherman You switch from spaces to tabs? eslint doesn't like that at all, hah

**Comment by zsherman** [comment]
@jsonmaur migrating off of vscode and haven't configured anything yet :/

File: themes/default/history.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from 'history'
+
+export default createBrowserHistory()
```
Comment (jsonmaur) [inline themes/default/history.js]: This will probably error during SSR. Maybe add `if (typeof window !== 'undefined')`?

Comment (zsherman) [reply themes/default/history.js]: yeah was about to test that

Comment (jsonmaur) [reply themes/default/history.js]: https://app.netlify.com/sites/gitdocs/deploys/5af36da70733d51a61e028a9

**Pull request merged**